tests/unit/components/sort-by-select/SortBySelect.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { vi } from 'vitest'
+
+import SortBySelect from '~/components/sort-by-select/SortBySelect'
+import { sortByFields } from '~/constants'
+
+const selectId = 'sort-by-select'
+const mockedSetSortBy = vi.fn()
+
+describe('SortBySelect component', () => {
+  it('should sort by newest if props sortBy state is newest', async () => {
```

Comment:
Delete all 'async' in this file, because you don't use await

Reply:
forgot to remove, thx